Sources/StreamChat/Controllers/EntityDatabaseObserver.swift

```diff
@@ -284,3 +363,87 @@ private extension ListChangeAggregator {
         return self
     }
 }
+
+/// Observes changes of a single entity specified using an `NSFetchRequest`in the provided `NSManagedObjectContext`.
+class BackgroundEntityDatabaseObserver<Item, DTO: NSManagedObject>: BackgroundDatabaseObserver<Item, DTO> {
+    var item: Item? {
+        rawItems.first
+    }
+
+    /// Called with the aggregated changes after the internal `NSFetchResultsController` calls `controllerDidChangeContent`
+    /// on its delegate.
+    private var listeners: [(EntityChange<Item>) -> Void] = []
+
+    /// Creates a new `ListObserver`.
+    ///
+    /// Please note that no updates are reported until you call `startUpdating`.
+    ///
+    /// - Important: ⚠️ Because the observer uses `NSFetchedResultsController` to observe the entity in the DB, it's required
+    /// that the provided `fetchRequest` has at lease one `NSSortDescriptor` specified.
+    ///
+    /// - Parameters:
+    ///   - context: The `NSManagedObjectContext` the observer observes.
+    ///   - fetchRequest: The `NSFetchRequest` that specifies the elements of the list.
+    ///   - itemCreator: A close the observe uses to convert DTO objects into Model objects.
```

Comment:
This is a bit unclear, probably "a closure the observer"?

Reply:
Copy paste 😂